### ISSUE TITLE
Customizing the admin interface

### ISSUE THREAD

**Issue by William-Hill** [SoftwareBrothers/admin-bro#164]
I would like to change the styling of the admin interface to match the look of the app I'm working on. I mainly want to change the background styling of the sidebar and main form areas. 

Is there a way to do this without writing my own custom dashboard?

**Comment by wojtek-krysiak** [comment]
yes - there is something like `theme` in branding options: https://softwarebrothers.github.io/admin-bro-dev/AdminBroOptions.html#branding but it is not well documented and has issues (#165). 

Here there are all available settings: https://github.com/SoftwareBrothers/admin-bro/blob/master/src/frontend/styles/variables.ts

**Comment by stealthman22** [comment]
Hello @wojtek-krysiak please the page doesn't exist anymore, and I would like to do some customization on the landing page and a few colour changes too.

I am trying really hard to follow the tutorial, but it doesn't cover everything 
Cheers

**Comment by udayanss** [comment]
@stealthman22 any luck?
